modules/afamqp/afamqp.c

```diff
@@ -448,8 +448,12 @@ afamqp_dd_socket_init(AMQPDestDriver *self)
 
             }
         }
+#ifdef SYSLOG_NG_HAVE_AMQP_SSL_SOCKET_SET_VERIFY_PEER
```

Comment:
IMHO it would be better if we would follow the same pattern as in case of OpenSSL, or GLib: hide compatibility issues behind a separated interface/file.

Reply:
Fixed thanks.